tests/route/conftest.py

```diff
@@ -6,3 +10,10 @@ def pytest_addoption(parser):
 
     route_group.addoption("--num_routes", action="store", default=10000, type=int,
                      help="Number of routes for add/delete")
+
+
+@pytest.fixture(scope='package', autouse=True)
+def prepare_arp_responder_conf(ptfhost):
+    arp_responder_conf = Template(open("../ansible/roles/test/templates/arp_responder.conf.j2").read())
+    ptfhost.copy(content=arp_responder_conf.render(arp_responder_args="--conf /tmp/from_t1.json"),
+                 dest="/etc/supervisor/conf.d/arp_responder.conf")
```

Comment:
I think we do not need to modify this `conftest.py` file since arp_responder is only used in static route tests. Adding these two lines under line 58 in `test_static_route.py` should work.
```
        ptfhost.host.options["variable_manager"].extra_vars.update({"arp_responder_args": "-e"})
        ptfhost.template(src="templates/arp_responder.conf.j2", dest="/etc/supervisor/conf.d/arp_responder.conf")

```

Reply:
I will update it to test if it is ok.
thanks

Reply:
hi @shi-su , I have updated it as your comments and test it. it passes. 
Please reveiw it again.